lib/zold/hungry_wallets.rb

```diff
@@ -0,0 +1,12 @@
+require 'delegate'
+
+module Zold
+  # Wallets decorator that adds missing wallets to the queue to be pulled later.
+  class HungryWallets < SimpleDelegator
+    # @todo #280:30min Add to the queue and return nil.
```

Comment:
@botanicus what `nil` is doing here? We _never_ return `nil`, it's a bad and totally unacceptable practice, please forget about `nil` entirely: https://www.yegor256.com/2014/05/13/why-null-is-bad.html